tests/network/expose.go

```diff
@@ -152,13 +123,14 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				Expect(executeVirtctlExposeCommand(vmiExposeArgs)).To(Succeed(), shouldExposeServiceViaVirtctl)
 
 				By(gettingValidatingClusterIP)
-				svc, err := getService(tcpVM.Namespace, serviceName)
+				svc, err := virtClient.CoreV1().Services(tcpVM.Namespace).Get(context.Background(), serviceName, k8smetav1.GetOptions{})
 				Expect(err).ToNot(HaveOccurred())
 				err = validateClusterIp(svc.Spec.ClusterIP, ipFamily)
 				Expect(err).ToNot(HaveOccurred())
 
 				By(iteratingClusterIPs)
-				runJobsAgainstService(svc, tcpVM.Namespace, job.NewHelloWorldJobTCP)
+				err = runJobAgainstService(svc, tcpVM.Namespace, job.NewHelloWorldJobTCP)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
nit: `Expect(runJobAgainstService(svc, tcpVM.Namespace, job.NewHelloWorldJobTCP)).To(Succeed())`